### PR TITLE
chore: updates to support deno_graph API changes

### PR DESCRIPTION
**Pull request by kitsonk**
This is what it takes to update to the deno_graph changes.

### CODE REVIEW

File: cli/tests/testdata/error_027_bundle_with_bare_import.ts.out [hunk 1]
```diff
@@ -1 +1 @@
-[WILDCARD]error: Relative import path "foo" not prefixed with / or ./ or ../ from "file:///[WILDCARD]/error_027_bundle_with_bare_import.ts"
+[WILDCARD]error: Relative import path "foo" not prefixed with / or ./ or ../
```
Comment (lucacasonato) [inline cli/tests/testdata/error_027_bundle_with_bare_import.ts.out:1]: This is a pretty serious degradation of error messages.

Comment (kitsonk) [reply cli/tests/testdata/error_027_bundle_with_bare_import.ts.out:1]: It was due to https://github.com/denoland/deno_graph/pull/75 by @nayeemrmn.

It was redundant in quite a few places, but not everywhere.  I don't know if @nayeemrmn had a plan in the places it was removed.

We could revert https://github.com/denoland/deno_graph/pull/75 and re-publish, or just move forward and open an issue to restore the information in the two scenarios it wasn't redundant.

Comment (lucacasonato) [reply cli/tests/testdata/error_027_bundle_with_bare_import.ts.out:1]: If you think we can re-add the info before 1.17, I am happy to merge as is.

Comment (bartlomieju) [reply cli/tests/testdata/error_027_bundle_with_bare_import.ts.out:1]: That might be addressed in https://github.com/denoland/deno/pull/12960 ?

**Review by bartlomieju — APPROVED** [review]
LGTM

File: cli/tests/testdata/error_027_bundle_with_bare_import.ts.out (hunk 1 above)
Comment (nayeemrmn) [reply cli/tests/testdata/error_027_bundle_with_bare_import.ts.out:1]: Yes #12960 restores it in precisely these two places 
image
 I don't know exactly why these weren't caught in #12604 and are now but I guess the code is cleaner.

**Review by lucacasonato — APPROVED** [review]
LGTM, under the condition that we land some form of https://github.com/denoland/deno/pull/12960 that fixes the regressed error messages.

**Review by dsherret — APPROVED** [review]
LGTM too.

**Pull request merged**